ansible/roles/master/templates/kubelet.j2

```diff
@@ -4,14 +4,11 @@
 # The address for the info server to serve on (set to 0.0.0.0 or "" for all interfaces)
 KUBELET_ADDRESS="--address=0.0.0.0"
 
-# The port for the info server to serve on
-# KUBELET_PORT="--port=10250"
-
 # You may leave this blank to use the actual hostname
-KUBELET_HOSTNAME="--hostname-override={{ inventory_hostname }}"
+KUBELET_HOSTNAME=""
```

Comment:
Is their a reason why we need to remove:

--hostname-override={{ inventory_hostname }}"